src/Hello.java

```diff
@@ -15,7 +15,7 @@ static String parser(String[] args) {
 
     public static void main (String[]args){
         String name = parser(args);
-        System.out.println("Hello " + name + "I am very exited to work at " + name);
+        System.out.println("Hello " + name + "I am very exited to work at " + name + "!");
```

Comment:
Bu  System.out.printf( "Hello %s %n I am very exited to work at %s !", name ,name); şu şekilde de yapılabilirdi